### ISSUE TITLE
pat-structure and horizontal toolbar

### ISSUE THREAD

**Issue by 1letter** [plone/mockup#1335]
Case:

1. horizontal toolbar
2. a lot of catalog metadatas and columns in the structure table

Problem One:
Toolbar of the Structure Table is Sticky but hidden if i scroll down

Problem Two:
Only with a fixed Taskbar on Windows, the Save Button of the Column Selection UI Panel is not accessible

Screen for better understanding
![Bildschirmfoto vom 2023-08-01 09-33-53](https://github.com/plone/mockup/assets/11072000/b7e0b91c-b2b7-402f-819e-2c577284ee30)

**Comment by 1letter** [comment]
@petschki 
I have rewrite the layout of popover in a custom project to a 2-column layout via css. it's a solution for the longlist problem. the save button goes to the top of the popover panel.
![Bildschirmfoto vom 2023-08-09 10-27-55](https://github.com/plone/mockup/assets/11072000/8be8dc74-f4a9-4ce1-ad1c-bc5450e3c25b)

```
body{
  
  &.plone-toolbar-top{

    .pat-structure{
      
      .navbar{
        top: 48px;
        left: 0px;
        
        #popover-structure-columns{
          
          min-width: 600px;
          
          > .popover-title{
            font-weight:bold;
            width:100%;
            display: block;
          }
          
          >.popover-content{
            display:flex;
            flex-direction: column;
            align-items: flex-start;
            padding:0 1rem;
            >label{
              order:1;
              margin-bottom:0.5rem;
            }
            >ul{
              column-count: 2;
              order:3;
              margin-bottom:0.5rem;
            }
            >button{
              order:2;
              margin-bottom:0.5rem;
            }
          }
        }
      }
      .popover{
        &.attribute-columns{
          ul>li>label{
            display: flex;
            align-items: flex-start;
          }
        }
      } 
    }
  }
```

**Comment by petschki** [comment]
Ah yes, that's nice ... do you want to update that in https://github.com/plone/mockup/blob/master/src/pat/structure/structure.scss ... see my latest changes at the bottom.

**Comment by 1letter** [comment]
@petschki  Yes, I will do this this afternoon.

**Comment by 1letter** [comment]
closed via #1339

**Comment by petschki** [comment]
just released in `plone.staticresources = 2.1.6`
